civictechprojects/views.py

```diff
@@ -248,7 +248,14 @@ def event_delete(request, event_id):
 
 
 def get_event(request, event_id):
-    event = Event.objects.get(id=event_id)
+    if event_id.isnumeric():
```

Comment:
I think there's a potential source of confusion if `id=15` or `id=slugname` works for the owner - who would presumably want to send out links to their own event, and suddenly a link that works for them would fail silently for everyone else. Would it be possible to make it so that if an event has a slug ID, it is _always_ redirected to that slug ID for owners and visitors alike (admins could still use both if we need that), so there's no possible copy-and-paste failure state?

Reply:
Good point, we need a robust way to handle canonical urls.

Reply:
Okay, I saw the trello ticket for canonical. Works for me.